pyproject.toml

```diff
@@ -83,3 +83,6 @@ target-version = ["py38", "py39", "py310", "py311", "py312"]
 [tool.coverage.run]
 branch = true
 relative_files = true
+
+[tool.pytype]
+inputs = ['src']
```

Comment:
I don't think we use pytype in the project. The dependency would be missing. If you want to use it, I think this would belong in a different PR/ticket.

Reply:
ooops, this was not meant to be in this PR sorry...